lib/lein.sh

```diff
@@ -20,39 +20,3 @@ function is_lein_2() {
 		return 1
 	fi
 }
-
-function install_rlwrap() {
-	local buildDir="${1:?}"
-	local cacheDir="${2:?}"
-
-	APT_CACHE_DIR="${cacheDir}/clojure-bp-apt/cache"
```

Comment:
This will leave stale files behind in the cache from previous versions of the buildpack (relevant since we've seen builds fail or be slow for some customers for other languages with bloated caches).

Is it worth adding an `rm -rf "${cacheDir}/clojure-bp-apt"` somewhere, similar to this:
https://github.com/heroku/heroku-buildpack-python/blob/b9c2dc02ca0ba459a4546108d57aba5e9abde5e3/lib/cache.sh#L149-L155

Reply:
Good catch, I would've added this later with the broader refactors, but without this record I would've probably forgotten. I'll add it in this PR like I did for Scala.